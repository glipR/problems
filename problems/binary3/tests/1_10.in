8796093022208 15054