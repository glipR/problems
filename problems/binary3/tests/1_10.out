244726426262548
